dynomite/trybuild-tests/fail/default-with-flatten.stderr

```diff
@@ -0,0 +1,5 @@
+error: If #[dynomite(flatten)] is used, no other dynomite attributes are allowed on the field
+ --> $DIR/default-with-flatten.rs:5:16
+  |
+5 |     #[dynomite(default, flatten)]
```

Comment:
Are there technical constraints for why we can't support this?

I see defaulting values when absent and flattening structures as orthogonal.

Reply:
This needs more design effort, I am not sure I'll be able to add support for `(default, flatten)` in this PR.

The PR is a draft, I want to propagate those generated `#[doc(hidden)]` methods to a `FromAttributes` trait to support
flattening on `HashMap<String, T: Attribute>` to be able to collect all the remaining fields of the object ([as it is in serde](https://serde.rs/attr-flatten.html#capture-additional-fields))

Reply:
Thanks for clarifying